examples/chrono_binding.cpp

```diff
@@ -0,0 +1,173 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <cassert>
+#include <string>
+#include <iostream>
+
+#if __cpp_lib_chrono >= 201907L
+
+///////////////////////////////
+/// sys_days binding as TEXT
+/// ///////////////////////////
+
+#include <chrono>
```

Comment:
Please include before the library feature test above.

Reply:
done